inc/sitemaps/class-sitemaps-renderer.php

```diff
@@ -339,6 +339,11 @@ protected function get_xsl_url() {
 			return home_url( 'main-sitemap.xsl' );
 		}
 
+		// Fallback in case the plugin url is on a different (sub)domain.
```

Comment:
Please add the specific reason why this is needed (cross-domain security problem)